src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.cs

```diff
@@ -255,14 +250,7 @@ public bool SetMaxThreads(int workerThreads, int ioCompletionThreads)
                     return false;
                 }
 
-                if (ThreadPool.UsePortableThreadPoolForIO)
-                {
-                    _legacy_maxIOCompletionThreads = (short)Math.Min(ioCompletionThreads, MaxPossibleThreadCount);
-                }
-                else
-                {
-                    ThreadPool.SetMaxIOCompletionThreads(ioCompletionThreads);
-                }
+                _legacy_maxIOCompletionThreads = (short)Math.Min(ioCompletionThreads, MaxPossibleThreadCount);
```

Comment:
I know the part was there before, but how do we know this doesn't exceed the bounds of a short?

I think someone will have thought of something, but it might be useful to create a comment on this.

Reply:
`MaxPossibleThreadCount` is `short.MaxValue`, so this will always fit.